### ISSUE TITLE
v9 middleware registration

### ISSUE THREAD

**Issue by infabo** [AOEpeople/TYPO3_Restler#39]
https://github.com/AOEpeople/TYPO3_Restler/blob/2bb4b29f9bfab6da57eb25f4958b5443311ba544/Configuration/RequestMiddlewares.php#L10

shouln't this be 'before' instead of 'after'?

**Comment by sven-carstens** [comment]
Only after the PageResolver a page id is known and the TSFE with the TypoScript Settings can be loaded.

**Comment by infabo** [comment]
The routes `/api` and `/api_explorer` are not known to the PageResolver thus leading into a 404. seams like a deadlock situation...

**Comment by StefanMasz** [comment]
@infabo should be fixed on the current feature-Branch. Please check.

**Comment by develth** [comment]
Maybe this is related to my issue:
I made some endpoints to my extbase extension. Creating Objects work - but no deletion. After checking the pid of the objects, they are all set to 0. In the ```restler_examples``` is the 'old' configuration, that sets an pageUid in the Controller. 
How is it done now? This way does not seems to exist anymore. Should that be done by that feature?

**Comment by sven-carstens** [comment]
Could you please open a separate issue with some more details?
Maybe the code inside your DELETE method?

**Comment by felixsemmler** [comment]
Please check the latest version of [restler](https://extensions.typo3.org/extension/restler) and [restler_examples](https://extensions.typo3.org/extension/restler_examples) if the bug is fixed for you. We did some changes and update the code for TYPO3V10.
